examples/example_use.ipynb

```diff
@@ -310,7 +310,7 @@
     "model_flows = r.get(base_url+model_name+'/flows',headers=api_headers)\n",
     "model_flows_json = model_flows.json()\n",
     "first_flow_id = model_flows_json[0]['id']\n",
-    "first_flow_name = model_flows_json[0]['name']\n",
+    "first_flow_name = model_flows_json[0]['flowable']\n",
```

Comment:
"name" key doesn't exist. "flowable" seems to be a good alternative and works with both USEEIOv2.0.1-411 and USEEIOv1.2 models.

Reply:
yes that's the correct field